deployment/helpers/solana.ts

```diff
@@ -128,28 +131,56 @@ export function getConnection(chain: SolanaChainInfo) {
   return connection;
 }
 
-/**
- * @todo Add priority fee instruction when configured to do so.
- */
-export async function ledgerSignAndSend(connection: Connection, instructions: TransactionInstruction[], signers: Keypair[]) {
+export type PriorityFeeConfiguration = {
+  lockedWritableAccounts: PublicKey[]
+  priorityFeePolicy: PriorityFeePolicy
+  defaultFee?: number
+}
+
+export async function ledgerSignAndSend(connection: Connection, 
+  instructions: TransactionInstruction[], 
+  signers: Keypair[],
+  priorityFeeConfig?: PriorityFeeConfiguration
+) {
   const deployerSigner = await getSigner();
 
   const deployerPk = new PublicKey(await deployerSigner.getAddress());
 
   const tx = new Transaction();
 
-  tx.add(...instructions);
+  if (priorityFeeConfig) {
+    const priorityFee = await getPrioritizationFee(connection, 
+      priorityFeeConfig.lockedWritableAccounts, 
+      priorityFeeConfig.priorityFeePolicy, 
+      priorityFeeConfig.defaultFee);
+    console.log(`Using priority fee policy: ${priorityFeeConfig.priorityFeePolicy}, selected fee: ${priorityFee} microLamports`);
+    const addPriorityFeeIx = ComputeBudgetProgram.setComputeUnitPrice({ 
+      microLamports: priorityFee
+    });
+    tx.add(addPriorityFeeIx);
+  }
 
-  const recentBlockHash = await connection.getLatestBlockhash();
+  tx.add(...instructions);
```

Comment:
Actually, I recommend adding the priority fee instruction at the end. We had txs fail when we didn't do so. I suspect this might have been a wormhole core program quirk but just in case, I'd follow the same pattern here.